src/index.js

```diff
@@ -57,6 +62,9 @@ export {
   DurationDisplay,
   TimeDivider,
   VolumeMenuButton,
+  VolumeBar,
+  VolumeLevel,
+  PopupButton,
```

Comment:
Do we need PopupButton? I don't think we're using it at the moment in Angelou

Reply:
No, we don't. And we don't need `VolumeLevel` either.